### PR TITLE
MEPTS-404: fixed C1 indicator

### PR DESCRIPTION
**Pull request by cfaife**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java [hunk 1]
```diff
@@ -742,20 +741,60 @@ public CohortDefinition getPatientsWhoInitiatedPreTarvDuringCurrentMonthAndScree
...
+
+    cd.setCompositionString("A2 AND TB");
+    //cd.setCompositionString("TB");
 
-    cd.setCompositionString("Pop AND TB");
 
     return cd;
   }
+  
+  private CohortDefinition tbScreening() {
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: @cfaife I suggest to change the name of this function to something more clear to what it is doing...
```suggestion
  private CohortDefinition getNumberOfPatientTbScreenedInFirstEncounter()() {
```

Comment (psmatsinhe) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: @cfaife And please move to tb queries or generic

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java [hunk 2]
```diff
@@ -742,20 +741,60 @@ public CohortDefinition getPatientsWhoInitiatedPreTarvDuringCurrentMonthAndScree
...
         map(
-            getPatientsWhoInitiatedPreTarvAtAfacilityDuringCurrentMonthC1(),
-            "startDate=${startDate},endDate=${endDate},location=${location}"));
-    cd.addSearch("TB", map(tb, mappings));
+            getPatientsWhoInitiatedPreTarvAtAfacilityDuringCurrentMonthA2(),
+            mappings));
+    cd.addSearch("TB", map(tbScreening(), mappings));
+
+    cd.setCompositionString("A2 AND TB");
+    //cd.setCompositionString("TB");
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: @cfaife I guess you forgot this one

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java (hunk 1 above)
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: @cfaife  What is the difference with this funtion 
**private CohortDefinition getPatientScreenedForTb()**

**Review by psmatsinhe — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java (hunk 2 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: removed the commented line

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java (hunk 1 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: this  function uses  obs_datetime while the  new  we added uses  encounter_datetime. And the other thing is: the  query we wrote  user concepts `yes`  and  `no`  in value coded and getPatientScreenedForTb ues  only  yes concept.

Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: done

**Pull request merged**